example/src/main/java/com/stripe/example/activity/SetupIntentActivity.java

```diff
@@ -141,7 +141,7 @@ private void onCreatedSetupIntent(@NonNull ResponseBody responseBody) {
     private void retrieveSetupIntent() {
         final Observable<SetupIntent> setupIntentObservable = Observable.fromCallable(
                 () -> mStripe.retrieveSetupIntentSynchronous(
-                        SetupIntentParams.createRetrieveParams(mClientSecret)));
+                        ConfirmSetupIntentParams.createRetrieveParams(mClientSecret)));
```

Comment:
I think we can completely get rid of "retrieve params" and just pass the client secret String around

Reply:
Yup, that was a bug I didn't catch during the refactor.